packages/email-components/src/components/Alert/Alert.scss

```diff
@@ -0,0 +1,71 @@
+@use 'theme';
+
+.Alert {
+    display: block;
+    font-size: theme.$font-size;
+    line-height: theme.$line-height;
+    font-family: theme.$font-family;
+}
+
+.Alert strong {
+    font-weight: theme.$emphasis-font-weight;
+}
+
+.Alert__table {
+    width: 100%;
+    border-radius: theme.$border-radius;
+}
+
+.Alert__icon {
+    width: theme.$icon-size;
+    padding: theme.$padding;
+    padding-right: 0;
+    font-size: theme.$icon-font-size;
+    vertical-align: top;
+}
+
+.Alert__iconWithBorder {
+    display: inline-block;
+    box-sizing: content-box;
+    width: theme.$icon-with-border-size;
+    height: theme.$icon-with-border-size;
+    font-weight: theme.$icon-with-border-font-weight;
+    font-size: theme.$icon-with-border-font-size;
+    line-height: theme.$icon-with-border-line-height;
+    text-align: center;
+    vertical-align: middle;
+    border: theme.$icon-with-border-border;
+    border-radius: theme.$icon-with-border-radius;
+}
+
+.Alert__content {
+    padding: theme.$padding;
+}
+
+.Alert--center {
```

Comment:
Modifiers of main Block element should come after the main Block styles, in this case after line 8.